.github/workflows/check-semantic-commits.yml

```diff
@@ -20,7 +20,10 @@ jobs:
           validateSingleCommit: true
           scopes: |
             core
-            database
+            drivers
+            pkg
+            build
+            ci
```

Comment:
`ci` and `build` are top level type under conventional commits (and in fact, chore is the one that's somewhat more non-standardized -- it's mentioned in conventional commits standard, but not mentioned / deprecated in Angular). so I'd tend to leave them out of valid scopes, and use, e.g., `ci: xyz` or `build: xyz` over `chore(ci): xyz` or `chore(build): xyz`.

https://www.conventionalcommits.org/en/v1.0.0/#specification
https://github.com/angular/angular/blob/22b96b9/CONTRIBUTING.md#-commit-message-guidelines

Reply:
Corrected